homeworks/homework2.ipynb

```diff
@@ -6,7 +6,7 @@
    "source": [
     "# INFO8010: Homework 2\n",
     "\n",
-    "Last week you learned how to program your first neural network starting from the very first principles of deep learning. If you managed to solve last week's assignment without any problems **congratulations!**. If that was not the case **don't worry**, here's a second assignment for you which you can use to get better at deep learning.\n",
+    "Last week you learned how to program your first neural network starting from the very first principles of deep learning. If you managed to solve last week's assignment without any problems **congratulations!** If that was not the case **don't worry**, here's a second assignment for you which you can use to get better at deep learning.\n",
```

Comment:
"Last week" -> "Two weeks ago" or "Last time" or "In the previous assignment"
"last week's assignment" -> "the first assignment"